app/src/main/kotlin/com/wire/android/ui/newauthentication/login/NewLoginScreen.kt

```diff
@@ -90,27 +91,37 @@ private fun LoginContent(
     loginEmailState: LoginEmailState,
     userIdentifierState: TextFieldState,
     onNextClicked: (() -> Unit) -> Unit,
-    navigate: (NavigationCommand) -> Unit
+    navigate: (NavigationCommand) -> Unit,
+    canNavigateBack: Boolean,
+    navigateBack: () -> Unit,
 ) {
-    NewLoginContainer {
+    NewLoginContainer(
+        header = {
+            NewLoginHeader(
+                title = {
+                    Icon(
+                        imageVector = ImageVector.vectorResource(id = R.drawable.ic_wire_logo),
+                        tint = MaterialTheme.colorScheme.onBackground,
+                        contentDescription = stringResource(id = R.string.content_description_welcome_wire_logo),
```

Comment:
as far as this icon "has no any interest for user"  (it's not clickable) you can set `contentDescription = null`, so TalkBack ignores it.